ports/geckolib/binding_tools/regen_atoms.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+import os
+
+
+def gnu_symbolify(source, ident):
+    return "_ZN" + str(len(source.CLASS)) + source.CLASS + str(len(ident)) + ident + "E"
+
+
+def msvc64_symbolify(source, ident):
+    return "?" + ident + "@" + source.CLASS + "@@2PEAV" + source.TYPE + "@@EA"
+
+
+def msvc32_symbolify(source, ident):
+    return "?" + ident + "@" + source.CLASS + "@@2PAV" + source.TYPE + "@@A"
```

Comment:
TBH, I have no idea about msvc manglings. Their mangling rule looks complicated, so I disassemble the object file and copy the pattern to here directly.

If the type is just pointers, I guess reusing this is probably fine, but I'm unconfident about that anyway.